src/figuresClasses.ts

```diff
@@ -1,19 +1,94 @@
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape: Shape;
+  color: Color;
 
+  getArea(): number
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
 
-}
+  color: Color;
+
+  constructor(
+    color: Color,
+    private a: number,
+    private b: number,
+    private c: number,
+  ) {
+    this.color = color;
```

Comment:
![image](https://github.com/mate-academy/ts_figures_classes/assets/96409563/31b01686-83cf-4ebe-9e9a-ecbb6ac4c7bb)

Reply:
From my side I think reasonable not to fix cases, because in documentation of Typescript we need to use modifiers private, public or protected:

https://www.typescriptlang.org/docs/handbook/2/classes.html#parameter-properties